### DISCUSSION TITLE
CaseReducer conformance

### DISCUSSION THREAD

**Discussion by easiwriter** [pointfreeco/swift-composable-architecture#3538] [Q&A]
I'm getting the error: 'ProjectListFeature.Destination' does not conform to protocol 'CaseReducer'

This is the relevant code:

<img width="956" alt="Screenshot 2024-12-18 at 14 29 25" src="https://github.com/user-attachments/assets/3532c1f3-47e0-4e82-a042-b1673430d133" />

In the CaseReducer documentation it says:

You should not conform to this protocol directly. Instead, the Reducer macro will add a conformance to enums.

So what am I missing?

**Comment by Alex293** [comment]
You need to pass the features not the features's states, drop .State in enumeration cases

**Reply by easiwriter** [reply]
That did it! Thanks for this.

I have tried using stack based navigation but the NavigationLink causes the compiler to fail - ran out of time. If I comment out the link it compiles OK. 

Keith
